src/jdaTemplate/app/src/main/java/Bot/App.java

```diff
@@ -0,0 +1,33 @@
+package Bot;
```

Comment:
In Java, package names should be `all.lowercase`.

Reply:
Thanks for pointing this out i kinda forgot the convention. I'll fix it